.github/ISSUE_TEMPLATE/20-documentation-issue.md

```diff
@@ -4,14 +4,34 @@ about: Use this template for documentation related issues
 
 ---
 
-<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>
+<em>Thanks so much for taking the time to file a documentation issue and even more thanks if you intend to contribute to updating it! Please do introduce yourself on our [Gwitter channel](https://gitter.im/tensorflow/docs) and let us know if you have any questions. As a side note, per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>
```

Comment:
I think for general docs issues, we should point people to the docs mailing list instead of chat since more people can help there: [docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs)

Also, a mention of the Docs contributor guide might encourage pull requests: https://www.tensorflow.org/community/contribute/docs